libs/application/api-template-utils/src/lib/utils/assign.ts

```diff
@@ -0,0 +1,55 @@
+import jwt from 'jsonwebtoken'
+import { SendMailOptions } from 'nodemailer'
+import { dedent } from 'ts-dedent'
+
+import { Application } from '@island.is/application/core'
+
+export const createAssignToken = (application: Application, secret: string) => {
+  const token = jwt.sign(
+    {
+      applicationId: application.id,
+    },
+    secret,
+    { expiresIn: 24 * 60 * 60 },
```

Comment:
so one day, do we want it longer? three days? should we indicate when it expires in the email?

Reply:
Had not given this any thought. We could also have it optional in the action, making it up to the template developer to set expiry time (if it should be other than 24h) and decide if it should be a part of the email.

Reply:
Will cover this in another PR